internal/usecases/liquidity_provider/set_trusted_account.go

```diff
@@ -0,0 +1,39 @@
+package liquidity_provider
+
+import (
+	"context"
+
+	"github.com/rsksmart/liquidity-provider-server/internal/entities"
+	"github.com/rsksmart/liquidity-provider-server/internal/entities/liquidity_provider"
+	"github.com/rsksmart/liquidity-provider-server/internal/usecases"
+)
+
+type SetTrustedAccountUseCase struct {
+	trustedAccountRepository liquidity_provider.TrustedAccountRepository
+	signer                   entities.Signer
+	hashFunc                 entities.HashFunction
+}
+
+func NewSetTrustedAccountUseCase(
+	trustedAccountRepository liquidity_provider.TrustedAccountRepository,
+	signer entities.Signer,
+	hashFunc entities.HashFunction,
+) *SetTrustedAccountUseCase {
+	return &SetTrustedAccountUseCase{
+		trustedAccountRepository: trustedAccountRepository,
+		signer:                   signer,
+		hashFunc:                 hashFunc,
+	}
+}
+
+func (useCase *SetTrustedAccountUseCase) Run(ctx context.Context, account liquidity_provider.TrustedAccountDetails) error {
+	_, err := usecases.SignConfiguration(usecases.SetTrustedAccountId, useCase.signer, useCase.hashFunc, account)
```

Comment:
this is also not using the result of the signature, this use case is incomplete.